app_module/src/main/java/com/hmos/compat/app/AlertDialog.java

```diff
@@ -0,0 +1,254 @@
+            negativeClickedListener = listener;
+            return this;
+        }
+
+        public AlertDialog create() {
+            alertDialog = new AlertDialog(this);
+            return alertDialog;
+        }
+
+        public void show() {
+            alertDialog.show();
+        }
+    }
+
+    private int getInflateLayout() {
+        return ResourceTable.Layout_alert_dialog_layout;
+    }
+
+    private void prepareDialogViewButtonDy(ComponentContainer rootLayout) {
+        DirectionalLayout buttonLayout = AlertDialog.getComponent(rootLayout, ResourceTable.Id_btnLayout);
+        if (builder.positiveButtonText != null) {
+            this.setButton(2, builder.positiveButtonText, builder.positiveClickedListener);
+        }
+        if (builder.negativeButtonText != null) {
+            this.setButton(1, builder.negativeButtonText, builder.negativeClickedListener);
+        }
+        if (builder.neutralButtonText != null) {
+            this.setButton(0, builder.neutralButtonText, builder.neutralClickedListener);
+        }
+        if (builder.positiveButton != null && buttonLayout != null) {
+            buttonLayout.addComponent(builder.positiveButton);
+        }
+
+        if (builder.negativeButton != null && buttonLayout != null) {
+            buttonLayout.addComponent(builder.negativeButton);
+        }
+
+        if (builder.neutralButton != null && buttonLayout != null) {
+            buttonLayout.addComponent(builder.neutralButton);
+        }
+    }
+
+    private void prepareDialogViewButton(ComponentContainer rootLayout, Builder builder) throws NullPointerException {
+        DirectionalLayout titleFrame = AlertDialog.getComponent(rootLayout, ResourceTable.Id_titleFrame);
+        Text tvTitle = AlertDialog.getComponent(rootLayout, ResourceTable.Id_title);
+        Text tvContent = AlertDialog.getComponent(rootLayout, ResourceTable.Id_content);
+        // Setup title and title frame
+        if (tvTitle != null) {
+            tvTitle.setTextAlignment(TextAlignment.START);
+            if (builder.title == null) {
+                if (titleFrame != null) {
+                    titleFrame.setVisibility(Component.HIDE);
+                }
+            } else {
+                tvTitle.setText(builder.title.toString());
+                if (titleFrame != null) {
+                    titleFrame.setVisibility(Component.VISIBLE);
+                }
+            }
+        }
+        if (tvContent != null) {
+            if (builder.content != null) {
+                tvContent.setText(builder.content.toString());
+                tvContent.setVisibility(Component.VISIBLE);
+            } else {
+                tvContent.setVisibility(Component.HIDE);
+            }
+        }
+
+        prepareDialogViewButtonDy(rootLayout);
+
+
+
+
+    }
+
+    /**  Negative button.
+     *
+     * @param root root.
+     *
+     * @param id   id
+     *
+     * @param <E>   E.
+     *
+     * @return returns.
```

Comment:
Incorrect alignment & Spacing...Please make the changes required..and check all the javadocs, methods and java files for similar issue...as this issue is persistant in few other places too..